src/meshapi/views/forms.py

```diff
@@ -195,7 +195,7 @@ def join_form(request):
     )
 
 
-def get_next_free_nn() -> int:
+def get_next_free_nn_from_old_installs() -> int:
```

Comment:
Why the name change? Maybe it would be better to have a separate function?

Reply:
`get_available_nn`